usaspending_api/references/tests/test_location.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+from django.conf import settings
+from model_mommy import mommy
+import pytest
+
+from usaspending_api.common.api_request_utils import GeoCompleteHandler
+from usaspending_api.references.models import Location
+
+
+@pytest.mark.django_db
+def test_location_reference_fill():
+    country_code = mommy.make('references.RefCountryCode', country_code="USA", country_name="United States", _fill_optional=True)
+    city_county_code = mommy.make(
+        'references.RefCityCountyCode', city_code="A", county_code="B", _fill_optional=True)
+    location = mommy.make(
+        'references.Location', location_country_code=country_code, location_city_code="A", location_county_code="B")
+    assert location.location_city_name == city_county_code.city_name
+    assert location.location_county_name == city_county_code.county_name
+    assert location.location_state_code == city_county_code.state_code
+    assert location.location_country_name == country_code.country_name
+
+
+@pytest.mark.django_db
+def test_geocomplete_scope():
```

Comment:
Yeah, I just put it in the Location tests because it felt "logical" to me, as it dealt with location objects. We could really put it either place. Feel free to move it later if you want :) But in the interest of time I'm just going to leave it here for now.